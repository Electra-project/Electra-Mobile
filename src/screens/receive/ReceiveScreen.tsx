import React from 'react'
import { StyleSheet, Text, View } from 'react-native'

const styles = StyleSheet.create({
  container: {
    alignItems: 'center',
    backgroundColor: '#fff',
    flex: 1,
    justifyContent: 'center'
  }
})

export default class ReceiveScreen extends React.Component {
  public render(): JSX.Element {
    return (
      <View style={styles.container}>
        <Text>Receive</Text>
      </View>
    )
  }
}
