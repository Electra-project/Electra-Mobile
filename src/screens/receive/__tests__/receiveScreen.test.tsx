import React from 'react'
import { shallow } from 'enzyme'

import ReceiveScreen from '../ReceiveScreen'

describe('ReceiveScreen', () => {
    it('renders correctly', () => {
        const wrapper = shallow(<ReceiveScreen />)
        expect(wrapper).toMatchSnapshot()
    })
})
