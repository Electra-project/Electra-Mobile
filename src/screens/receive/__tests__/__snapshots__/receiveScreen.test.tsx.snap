// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReceiveScreen renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ReceiveScreen />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
      >
        Receive
      </Text>,
      "style": Object {
        "alignItems": "center",
        "backgroundColor": "#fff",
        "flex": 1,
        "justifyContent": "center",
      },
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "accessible": true,
        "allowFontScaling": true,
        "children": "Receive",
        "ellipsizeMode": "tail",
      },
      "ref": null,
      "rendered": "Receive",
      "type": [Function],
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
        >
          Receive
        </Text>,
        "style": Object {
          "alignItems": "center",
          "backgroundColor": "#fff",
          "flex": 1,
          "justifyContent": "center",
        },
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "accessible": true,
          "allowFontScaling": true,
          "children": "Receive",
          "ellipsizeMode": "tail",
        },
        "ref": null,
        "rendered": "Receive",
        "type": [Function],
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
